packages/components/src/docs/General/1-Introduction.mdx

```diff
@@ -42,3 +44,19 @@ Solid Components are designed to offer high reliability and easy maintainability
 - **Thorough testing**: High test coverage with E2E-Tests via Playwright (<a href={`https://github.com/microsoft/playwright/releases/tag/v${playwrightVersion}`} target="\_blank">tested Browser versions</a>) and Visual Regression Tests via Chromatic ([current Browser versions](https://www.chromatic.com/docs/infrastructure-release-notes)).
 
 Our emphasis is on creating a solid foundation of reliable, maintainable components that balance necessary features with ease of upkeep.
+
+## Structure
+
+### Styles & Components
+
+Our library offers reusable Styles and Components, each having their own technology and purpose:
+
+- **Styles**: These are standalone CSS files. They don't provide any logic and are often non-interactive.
+- **Components**: These are Web Components. They are often more complex than Styles, and could feature reactivity, state, multiple slots, properties, and more.
```

Comment:
I would have re-ordered Styles and Components and styles here since Components is sort of the "main" product we want to highlight.

Reply:
I wanted to go from `less-complicated` too `complicated`. :)

Reply:
That makes sense as well!